<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Native Speaker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>321</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_learning_models">
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_learning_models">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Running models learning</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_learning_models"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>20</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Models list</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_prev_models">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Prev Page</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="commandLinkButton_next_models">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Next Page</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolButton_refresh">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_act_page">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QToolBox" name="models_list">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabSpacing">
        <number>6</number>
       </property>
       <widget class="QWidget" name="page1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>389</width>
          <height>338</height>
         </rect>
        </property>
        <attribute name="label">
         <string/>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>27</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRead_Configuration_File"/>
    <addaction name="actionSave_Configuration_File"/>
    <addaction name="actionAdd_Configuration_Parameter"/>
   </widget>
   <widget class="QMenu" name="menuModele">
    <property name="title">
     <string>Models</string>
    </property>
    <addaction name="action_CreateModel"/>
    <addaction name="action_ModelLearning"/>
   </widget>
   <widget class="QMenu" name="menuRozpoznawanie">
    <property name="title">
     <string>Recognition</string>
    </property>
    <addaction name="actionVerification"/>
   </widget>
   <widget class="QMenu" name="menuBaza_pr_bek">
    <property name="title">
     <string>Sample Base</string>
    </property>
    <addaction name="actionView_sample_base"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuModele"/>
   <addaction name="menuRozpoznawanie"/>
   <addaction name="menuBaza_pr_bek"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_CreateModel">
   <property name="text">
    <string>Model Creation</string>
   </property>
  </action>
  <action name="action_ModelLearning">
   <property name="text">
    <string>Model Learning</string>
   </property>
  </action>
  <action name="action_RecordFromFileIdentification">
   <property name="text">
    <string>Record from file</string>
   </property>
  </action>
  <action name="actionRead_Configuration_File">
   <property name="text">
    <string>Read Configuration File</string>
   </property>
  </action>
  <action name="actionSave_Configuration_File">
   <property name="text">
    <string>Save Configuration File</string>
   </property>
  </action>
  <action name="actionAdd_Configuration_Parameter">
   <property name="text">
    <string>Add Configuration Parameter</string>
   </property>
  </action>
  <action name="actionVerification">
   <property name="text">
    <string>Verification</string>
   </property>
  </action>
  <action name="actionView_sample_base">
   <property name="text">
    <string>View sample base</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>commandLinkButton_prev_models</tabstop>
  <tabstop>commandLinkButton_next_models</tabstop>
  <tabstop>toolButton_refresh</tabstop>
  <tabstop>lineEdit_act_page</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
